jobrunner/run.py

```diff
@@ -20,7 +20,7 @@
     finalise_job,
     job_still_running,
     kill_job,
-    start_job,
+    start_job, list_outputs_from_action,
```

Comment:
It looks like this repo has isort configured but not enforced by CI, could you pop this on the line before `start_job`?

Reply:
Will do. Thanks.